modules/account-lib/src/coin/xrp/iface.ts

```diff
@@ -0,0 +1,33 @@
+export interface SignedTransaction {
+  signedTransaction: string;
+  id: string;
+}
+
+export interface ApiMemo {
+  MemoData?: string;
+  MemoType?: string;
+  MemoFormat?: string;
+}
+
+export interface TxJSON {
+  account: string;
+  type: string;
+  memos?: { Memo: ApiMemo }[];
+  flags?: number;
+  fulfillment?: string;
+  amount?: string;
+  destination?: string;
+  fee?: string;
+  sequence?: number;
+  lastLedgerSequence?: number;
+  domain?: string;
+  setFlag?: number;
+  messageKey?: string;
+  destinationTag?: number;
```

Comment:
`destinationTag` is another field that can be abstracted to make it more generic.
Please create an interface like in the example below and use it here:
```
interface paymentId {
	name: string;    // "Destination Tag", "Memo Id", "Transfer Id"
	keyword: string; // "dt", "memoId", "transferId"
	value: string;   // number || text || hash
```